.github/workflows/consecutive.yml

```diff
@@ -63,54 +98,55 @@ jobs:
         with:
           folder-name: report/allure-results
 
-  node-contracts:
-    runs-on: neon-hosted
-    timeout-minutes: 361
-    if: always()
+  openzeppelin:
+    timeout-minutes: 360
```

Comment:
it's the default value, the same as we omit the timeout-minutes parameter at all.
I noticed a couple of time that GHA job got frozen and, even I don't know how 'software from Ivanovo' works internally, just tried to set a limit a bit bigger than possible for a job

Reply:
In self-hosted runner job can be run more 6 hours (up to 24).